pallets/human_swap/README.md

```diff
@@ -1,3 +1,36 @@
 # Human Swap Pallet
 
-Swaps that require human interaction(potentially automated on the human side), these humans(liquidity providers) are willing to exchange their own assets at a rate often higher than that of a configured price oracle, where one of the assets in the trading pair is a foreign(off-chain) asset, meaning that its completion depends on real world actions that when confirmed release the scrowed funds of the on-chain asset being traded.
+For swaps that require human interaction due to one of the trading pairs being a foreign asset(an asset that exists off-chain).  
+Users of this pallet specify a human they want to trade with based on the convenience of their rates, the party sending cryptocurrency
+gets its funds locked in a scrow and the other receives the information necesary to conduct the off-chain payment.
+
+![swap_flow](https://user-images.githubusercontent.com/1329925/115603536-90c2ed80-a2e0-11eb-810a-ebbe464d115a.png)
+
+This pallet tracks the state of the swaps which are updated based on actions form the user and the human involved. In the image each of the actions for either a `swap_in` or a `swap_out` changes the state of the overal swap which looks something like
+```rust
+struct Swap {
+  human: AccountId,
+  type: SwapType,
+  rate: PairRate,
+  amount: Balance,
```

Comment:
Shouldn't we be storing the swap-recipient also? especially in cash-out we need this account to reserve Usdv and then transfer to LP?

Reply:
I was thinking we already have that info since it makes sense to have the swap information indexed by the user account and swap id as keys of the storage, i.e. `(UserAccountId, SwapId) -> Swap` so we can query the swaps of a user or a specific one. Actually the `SwapId` there can be just a nonce, like the one you would get from `System::account_nonce` and what we call `SwapId` be the combination of both the user account and the nonce.